modules/migration-network-traffic-redirection-strategies.adoc

```diff
@@ -0,0 +1,50 @@
+
+* Applications are deployed on the {product-title} {product-version} cluster. The original application pods are still running on the source cluster.
+* Each application has a route that contains the source cluster hostname.
+* The route with the source cluster hostname contains a CA certificate.
+* If necessary for securing connections made through the source cluster's DNS domain, the {product-title} {product-version} router default certificate includes the source cluster wildcard Subject Alternative Name.
+
+Review the following strategies and select the one that meets your objectives.
+
+* Redirecting all network traffic for all applications at the same time
++
+Change the source cluster wildcard DNS record to point to the {product-title} {product-version} router's virtual IP address (VIP).
++
+This method is suitable for simple applications or small-scale migrations.
+
+* Redirecting  network traffic for individual applications
++
+Create a DNS record for each application with the source cluster hostname pointing to the {product-title} {product-version} router's VIP. This DNS record takes precedence over the source cluster wildcard DNS record.
+
+* Gradual redirection of network traffic for individual applications
+
+. Create a VIP or proxy with two back ends, the source cluster router's VIP and the {product-title} {product-version} router's VIP, for each application.
+
+. During migration, create a DNS record for each application with the source cluster hostname pointing to the VIP or proxy. This record takes precedence over the source cluster wildcard DNS record.
+
+. Configure the proxy entry for the application to route a percentage of the traffic to the {product-title} {product-version} VIP and the rest of the traffic to the source cluster router VIP.
+
+. Gradually increase the percentage of traffic that you route to the {product-title} {product-version} VIP.
+
+* User-based redirection of traffic for individual applications
++
+This method employs filtering TCP/IP headers of user requests to redirect network traffic for predefined groups of users. This allows you to test the redirection process on specific populations of users before redirecting the entire network traffic.
+
+. Create a proxy with two backends, the source cluster router's VIP and the {product-title} {product-version} router's VIP, for each application.
+
+. During migration, create a DNS record for each application with the source cluster hostname pointing to the proxy. This record takes precedence over the source cluster wildcard DNS record.
+
+. Configure the proxy entry for the application to route traffic matching a given header pattern, such as `test customers`, to the {product-title} {product-version} router's VIP and the rest of the traffic to the source cluster VIP.
+
+. Move traffic to the {product-title} {product-version} VIP in stages until all the traffic is on the {product-title} {product-version} VIP.
```

Comment:
"Move" is not quite the right word. The traffic is not being moved. It is being redirected by filtering on TCP/IP headers, so you should say this.